tests/legacy_comparison/archive_config.py

```diff
@@ -0,0 +1,12 @@
+from response_comparisons import compare_status
+
+res_comparisons = [ compare_status ]
+text_comparisons = []
+html_comparisons = []
+
+def ng_id_to_url_fn(id:str)->str:
+    return 'http://localhost:5000/archive/' + id
```

Comment:
Is this intentional?

Reply:
Yes. Is it a good idea to hard code that URL prefix? No, it should be a command line arg or something.

Reply:
For this function, it would probably be ok to use `url_for('archive', ...)`. But not a huge deal as it's in a test.